arch/arm/boot/dts/nintendo3ds_ktr.dts

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Support for:
+ *   - New Nintendo 3DS    (KTR-001)
+ *   - New Nintendo 3DS XL (RED-001)
```

Comment:
Oh this is missing "new 2ds xl."  Was there a non-xl new 2ds?

Reply:
Is JAN the model number for the new 2ds xl?

Reply:
nope, the only 2ds model with the "new" hardware ever released was an XL model, and yes it's JAN-001